graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -27,14 +31,15 @@ def rename_query(
     any occurrences of type Y in the input query ast will be renamed to X.
 
     All type names (including ones in type coercions), as well as root vertex fields (fields
```

Comment:
This docstring mentions quite a bit of stuff about types, but based on the `Args` section it seems like the function will also rename fields? It's quite surprising to read the entire docstring up to the `Args` section and not see a mention of field renaming (other than root vertex fields, which are types). Even the one-liner at the top of the docstring only mentions types.

Consider touching up the rest of this docstring if the function's contract is indeed changing to include field renaming as well.